pyanaconda/ui/gui/spokes/source.py

```diff
@@ -494,6 +499,11 @@ def _method_changed(self):
 
             if old_source.partition in self.storage.config.protectedDevSpecs:
                 self.storage.config.protectedDevSpecs.remove(old_source.partition)
+                if old_source.partition.type == "partition" and old_source.partition.isLogical:
+                    # mark the extended partition as not protected too
+                    extended = self.storage.devicetree.getDeviceByPath(old_source.partition.disk.format.extendedPartition.path)
+                    extended.protected = False
+                    self.storage.config.protectedDevSpecs.remove(extended.name)
```

Comment:
This should only happen if this is the last protected logical partition from the extended partition which is being made unprotected.